src/view/components/microbit/MicrobitSimulator.tsx

```diff
@@ -100,14 +112,9 @@ export class MicrobitSimulator extends React.Component<any, IState> {
         return (
             <div className="simulator">
                 <div className="file-selector">
-                    <Dropdown
-                        label={"file-dropdown"}
-                        styleLabel={"dropdown"}
-                        lastChosen={this.state.running_file}
-                        width={300}
-                        textOptions={this.state.active_editors}
-                        onBlur={this.onSelectFile}
-                    />
+                    {this.state.running_file && this.state.play_button
+                        ? CONSTANTS.CURRENTLY_RUNNING(this.state.running_file)
+                        : null}

```

Comment:
Currently the text just pops out of nowhere and also looks a bit out of place. Perhaps play around with the font style and/or think about different ways to expose this information?

Reply:
I have made a set a height and a box to address this feedback :)